src/components/Animator/AnimatorComponent.scss

```diff
@@ -79,6 +79,7 @@
         }
 
         .playback-numeric-control {
+            margin-right: 10px;
```

Comment:
looks like there will be an extra space on Chrome after this fix:
<img src="https://user-images.githubusercontent.com/43841102/151287607-2e8ecc6e-ed9e-4e16-b6b4-6b93b28b7b46.png" width="300">

Reply:
I cannot eliminate the space difference between Chrome and Firefox.
So far I only optimize the scss and make sure there is an acceptable space there in both browsers.
Would it be OK or other better solution?

Reply:
@YuHsuan-Hwang I follow your suggestion to check and fix the width of "Frame Rate" dropdown, now I hope the scss looks better on both Chrome and Firefox!

Reply:
@acdo2002 looks good to me now! Do you think we can change `.bp3-input-group { width: 35px; }` back to 40px?

Reply:
Yes, I change it back to 40px and optimize bp3-html-select to 115px.